components/uniset/src/uniset.rs

```diff
@@ -213,18 +233,19 @@ impl UnicodeSet {
     /// in a single `O(1)` step
     ///
     /// Example: `contains_range(&0, &10)`
-    pub fn contains_range(&self, start: u32, end: u32) -> Result<bool, Box<dyn Error>> {
+    pub fn contains_range(&self, start: char, end: char) -> Result<bool, Box<dyn Error>> {
```

Comment:
It would be nice if this method were infallible.

According to https://doc.rust-lang.org/std/ops/struct.Range.html:

> The range ... is empty unless start < end.

Should we adopt the same convention here?

Reply:
agreed and changed